assets/sass/style.scss

```diff
@@ -7,6 +7,18 @@
   border-style: none;
 }
 
+.pointer {
+  cursor: pointer;
+}
+
+.bottom-bar {
+  border-top: solid 1px #CCC;
+}
+
+.custom-select {
+  background: #FFF;
```

Comment:
I found this difference between the bootstrap site styline and reef-pi styling for .custom-select. 
-webkit-appearance: none;

I don't think we should need to change with background: #FFF; I would prefer to find root cause of losing -webkit-appearance:none

Reply:
Based on https://getbootstrap.com/docs/4.1/getting-started/webpack/ I think it needs postcss-loader

Reply:
@Ranthalion merging this for the time being, if you find this is problematic or annoying, happy to revisit this

Reply:
I'll add post css loader so.